igor.dzetcyna/first/task 1/array.rb

```diff
@@ -0,0 +1,8 @@
+# Заповнити випадковими числами від -10 до 10 двовимірний масив 3х3 елементів
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#english-comments
Please write all comments in english